test/suite/test_hs01.py

```diff
@@ -35,7 +35,7 @@
 # Smoke tests to ensure history store tables are working.
```

Comment:
I know this isn't your test originally but can we update these comments to be a bit more specific, describe what the test is doing in this with more detail. 

Then on line 37 can you delete the small cache comment? We're working under the assumption that cache size is not important in this test.

Reply:
The comment on line 35 should be more specific, i.e.: "Test that update and modify operations are durable across crash and recovery. Additionally test that checkpoint inserts content into the history store."

We don't actually read behind any values here so it's almost not a history store test.